src/main/java/org/minions/devfund/lourdes/battleship/Ocean.java

```diff
@@ -0,0 +1,127 @@
+package org.minions.devfund.lourdes.battleship;
+
+import java.util.Random;
+
+public class Ocean {
+    private Ship[][] ship;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+    private String[] shipType = {"Battleship", "BattleCruiser", "Cruiser", "LightCruiser", "Submarine"};
+
+    public Ocean() {
+        this.ship = new Ship[20][20];
+        this.shotsFired = 0;
+        this.hitCount = 0;
+        this.shipsSunk = 0;
+        for (int i = 0; i < ship.length; i++) {
+            for (int j = 0; j < ship[i].length; j++) {
+                ship[i][j] = new EmptySea();
+            }
+        }
+    }
+
+    public void placeAllShipsRandomly() {
+        Random r = new Random();
+        int row = r.nextInt((0 - ship.length) + 1);
+        int column = r.nextInt((0 - ship.length) + 1);
+        boolean horizontal = r.nextBoolean();
+        ShipFactory shipFactory = new ShipFactory();
+        Ship newShep = shipFactory.createShip("Battleship");
+        newShep.placeShipAt(row, column, horizontal, this);
+    }
+
+    public boolean isOccupied(int row, int column) {
+        return !ship[row][column].getShipType().equals("empty");
+    }
+
+    public boolean shootAt(int row, int column) {
+        if (isOccupied(row, column) &&
+                getShipArray()[row][column].toString().equals("S") &&
+                getShipArray()[row][column].shootAt(row, column)) {
+            hitCount++;
+            if (getShipArray()[row][column].isSunk()) shipsSunk++;
+            return true;
+        }
+        getShipArray()[row][column].shootAt(row, column);
+        shotsFired++;
+        return false;
+    }
+
+    public void print() {
+        printColumnPosition();
+        for (int i = 0; i < ship.length; i++) {
+            System.out.println("");
+            System.out.print(i);
+            for (int j = 0; j < ship.length; j++) {
+                if (ship[i][j].toString().equals("S") && !ship[i][j].hit[ship[i][j].getHitIndex(i, j)]) {
+                    System.out.print(" . ");
+                } else {
+                    System.out.print(" " + ship[i][j] + " ");
+
+                }
+
+            }
+            System.out.println("");
+        }
+        System.out.println("");
+
+    }
+
+    public void printColumnPosition() {
+        StringBuilder arrayPosition = new StringBuilder(" ");
+        for (int i = 0; i < ship.length; i++) {
+            if (i < 10) {
+                arrayPosition.append("0");
+            }
+            arrayPosition.append(i);
+            arrayPosition.append(" ");
+
+        }
+        System.out.print(arrayPosition);
+    }
+
+    public static void main(String arg[]) {
+        Ocean ocean = new Ocean();
+        Ship a = new Submarine();
+        a.placeShipAt(3, 3, true, ocean);
+//        ocean.shootAt(3,4);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)